monai/networks/layers/spatial_transforms.py

```diff
@@ -559,7 +559,7 @@ def forward(
                 affine=theta,
                 src_size=src_size[2:],
                 dst_size=dst_size[2:],
-                align_corners=self.align_corners,
+                align_corners=False,
```

Comment:
Hi @wyli, could you please help me understand here why we need to hard-code this `align_corners` to `False`? Thanks!

Reply:
yes, this pytorch function already includes the normalization, if we do it here again, it generates the wrong results... https://github.com/Project-MONAI/MONAI/blob/5cb680ff29ab1a482767b3b29e60cea3cb0d8ab3/monai/networks/layers/spatial_transforms.py#L577

Reply:
the current dev branch of this argument is wrong when `normalized=False`, it effectively always generates the same result using `align_corners=False` even though the default is `align_corners=True`. this was confusing and I didn't realise this issue earlier.